src/libraries/System.Text.Json/gen/Resources/xlf/Strings.cs.xlf

```diff
@@ -3,15 +3,25 @@
   <file datatype="xml" source-language="en" target-language="cs" original="../Strings.resx">
     <body>
       <trans-unit id="ContextClassesMustBePartialMessageFormat">
-        <source>Derived 'JsonSerializerContext' type '{0}' specifies JSON-serializable types. The type and all containing types must be made partial to kick off source generation.</source>
-        <target state="translated">Odvozený typ JsonSerializerContext {0} určuje typy serializovatelné na JSON. Typ a všechny obsahující typy musí být částečné, aby se zahájilo generování zdroje.</target>
+        <source>Derived 'JsonSerializerContext' type '{0}' and all containing types must be made partial to kick off source generation.</source>
```

Comment:
If we do backport this to `release/6.0` now, isn't there a risk of translations going out of sync?

Reply:
We have a 1 week SLA for translations in `main`.  We have enough time to backport to `6.0`.  That backport will need to be tracked with an issue so we don't forget.